docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -141,6 +141,9 @@ public class CustomNamedHttpSecPolicy implements HttpSecurityPolicy {
 ----
 <1> Named HTTP Security policy will only be applied to requests matched by the `application.properties` path matching rules.
 
+TIP: If you need to inject the `jakarta.ws.rs.container.ResourceInfo` bean into the custom policy,
```

Comment:
I'm not sure this TIP is necessary, it appears to be a specific case and it is not clear from this Tip what is important about `ResourceInfo` as opposed to other JAX-RS injections and what it is to do with the policy configuration. I guess we can expand it into a dedicated section, but should we, it is not clear

Reply:
Yes, I agree with all above. TBH I only added it to avoid you asking me for it. I'll drop it.

Reply:
It's gone, thanks.